Summers_James/detail.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponseRedirect
+from django_mako_plus import view_function, jscontext
+from django import forms
+import re
+from catalog import models as cmod
+import math
+from django.conf import settings
+from datetime import datetime, timezone
+from django.shortcuts import render
+
+
+@view_function
+def process_request(request, prod_id=None):
+    product = cmod.Product.objects.filter(id=prod_id)[0]
+    cName = product.category
+
+    underline = int(cmod.Category.objects.filter(name=cName)[0].id)
+
+    myCategories = cmod.Category.objects.all()
+
+    images = product.image_urls()
+    path = product.image_url()
+    myList = []
+    for im in images:
+        myList.append(str(im.filename))
+
+#put current product being viewed into the request.lastFive
+    print("My PIDPIDPID " + str(product.id))
```

Comment:
Removing any print statements before merging to master

Reply:
Or maybe I should at least comment out. But yes. I don't want to blow out a bunch of stuff on the users console. :)